src/pages/cart/cart.jsx

```diff
@@ -0,0 +1,28 @@
+import "./cart.css"
+import { Footer, Navbar } from "../../components"
+import { CartCard } from "../../components/card/cart-card/cart-card"
+import { useProduct } from "../../contexts/product-context";
+import { useDocumentTitle } from "../../custom-hooks/useDocumentTitle"
+import { PriceSection } from "./sub-components/price-section";
+
+
+const Cart = () => {
+    useDocumentTitle("Cart")
+    const {cart} = useProduct();
+    return (
+        <>
+        <Navbar />
```

Comment:
I think the `Navbar` and `Footer` component is the same for the whole app, so instead of importing them in each page file. You can import it only in `App.js` and keep it outside the `Routes`.

Reply:
I will not be using the `navbar` and `footer` for the sign-in and login page. That's why I am not adding it in the `App.js` file.

Reply:
Thank you so much, Himanshu. This was a great approach👍. I have made the changes.